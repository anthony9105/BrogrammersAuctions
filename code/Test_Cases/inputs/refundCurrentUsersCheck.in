login adminUser1
refund user1234 user5678 500
