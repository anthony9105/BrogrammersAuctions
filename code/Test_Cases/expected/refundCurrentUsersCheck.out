Login successful
Error. Buyer is no longer a current user 
Error. Seller is no longer a current user 
